pipeline/dwi_masking.py

```diff
@@ -217,13 +217,13 @@ def normalize(b0_resampled):
     output_file = path.join(path.dirname(input_file), output_name)
     img = nib.load(b0_resampled)
     imgU16 = img.get_data().astype(np.float32)
-    p = np.percentile(imgU16, 99)
+    p = np.percentile(imgU16, percentile)
     data = imgU16 / p
     data[data > 1] = 1
     data[data < 0] = 0
     image_dwi = nib.Nifti1Image(data, img.affine, img.header)
     nib.save(image_dwi, output_file)
-    return output_file
+    data_n.append(output_file)
 
```

Comment:
@SenthilCaesar , what's the point of appending to `data_n`, if you are not returning?

Reply:
data_n is a Manager() object . Below resource might be helpful

https://stackoverflow.com/questions/9436757/how-does-multiprocessing-manager-work-in-python